README.md

```diff
@@ -142,6 +163,25 @@ hosts=host_list_provider(hosts=["some.host", "172.100.100.20"], ssh_config=ssh)
 ...
 ```
 
+### Use ssh-agent for ssh and scp connections
+By default, the ssh and scp connections in crashd use the default ssh-agent process pointed by the `SSH_AGENT_PID` and `SSH_AUTH_SOCK` environment variables.
+
+```bash
+# starts a ssh-agent
+$> eval "$(ssh-agent -s)"
+$> ssh-add /foo/bar_rsa
+...
+# ssh/scp connections in the diagnotics.crsh script will use the above ssh-agent
+$> crashd run diagnostics.crsh 
+```
+
```

Comment:
Add a small sentence or two about using ssh-agent for passphrase-protected keys.